scripts/distribute-version.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+import time
+
+
+def push_pod(name):
+    while True:
+        status = subprocess.run('pod repo update && pod trunk push ' + name + '.podspec', shell=True)
```

Comment:
Technically this can sometimes fail, for example, when the token is revoked, will CircleCI bill us for a couple of months of an endless cycle? 😁